documentation/guides/tutorials/make-a-zio-app-configurable/build.sbt

```diff
@@ -1,32 +0,0 @@
-import BuildHelper._
```

Comment:
@kyri-petrou 
Have you checked if the documentation website can include this tutorial after removing the file?

Reply:
If I remember correctly, this project's `mdoc` commands, compile the document and place it into the website documentation. Removing this project's build file won't generate related documentation materials for the zio website.

Reply:
The config from this `build.sbt` file was moved into the main `build.sbt` file though.

I also run `docs/mdoc` and it was definitely being included, although I didn't check the generated artifacts. I'll double check just in case

Reply:
@khajavi I just double-checked and this module's documentation is included in the website artifacts (at least the ones generated by mdoc).

Reply:
Thank you for the check!